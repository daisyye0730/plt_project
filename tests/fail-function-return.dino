int gcd(int a, int b) {
  return "hi";
}

int main() {
    int a;
    string str;
    str = gcd(4, 5);
    return 0;
}