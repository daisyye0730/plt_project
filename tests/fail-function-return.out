Fatal error: exception Failure("return gives string expected int in hi")
